packages/atomic/scripts/split-locales.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+const localesJSONData = fs.readFileSync('src/locales.json');
```

Comment:
Instead of `readSync` and the callback for `fs.mkdir`, you could use node promisify, it should make all of this easier to read.

Reply:
didn't know that was a thing and I like it very much 🙌